test/integration/provider/gcp.go

```diff
@@ -110,92 +122,49 @@ func getAvailableVolumes(ctx context.Context, svc *compute.Service, tagName stri
 	}
 
 	zone := providerSpec.Zone
-	req := svc.Disks.List(project, zone)
-
-	//check whether these tags are present on the disks or not
-	if err := req.Pages(ctx, func(page *compute.DiskList) error {
-		for _, disk := range page.Items {
-			for key, value := range disk.Labels {
-
-				// #the disks didn't have any tag, check after getting access
-				if strings.Contains(key, tagName) && strings.Contains(value, tagValue) {
-					fmt.Printf("%s", disk.Name)
-					availVolID = append(availVolID, disk.Name)
-
-					//delete the disk
-					deleteVolume(ctx, svc, project, zone, disk.Name)
-					break
-				}
-			}
+	// req := svc.Disks.List(project, zone).Filter("status = READY")
+
+	// //check whether these tags are present on the disks or not
+	// if err := req.Pages(ctx, func(page *compute.DiskList) error {
+	// 	for _, disk := range page.Items {
+	// 		for key, value := range disk.Labels {
+
+	// 			// #the disks didn't have any tag, check after getting access
+	// 			if strings.Contains(key, tagName) && strings.Contains(value, tagValue) {
+	// 				fmt.Printf("%s", disk.Name)
+	// 				availVolID = append(availVolID, disk.Name)
+
+	// 				//delete the disk
+	// 				//deleteVolume(ctx, svc, project, zone, disk.Name)
+	// 				break
+	// 			}
+	// 		}
+	// 	}
+	// 	return nil
+	// }); err != nil {
+	// 	return availVolID, err
+	// }
```

Comment:
Can be removed!

Reply:
Yeah I just kept it here to , because the commented logic uses tags to find volume but now I am using the volume names to find volumes. I get volume names from machine names directly as they both have same name.

Reply:
I will delete it since you are fine with the new logic now